docs/user-guide/content-web-toolbar.md

```diff
@@ -0,0 +1,13 @@
+# Web Page Toolbar
+
+As soon as an web page content is added, the new toolbar appears at the top of the content itself:
```

Comment:
As soon as a web page content is added, the toolbar appears on top of the content itself: